web-console/knikubevirt/vm-details/vm-boot-order/vm-boot-order.md

```diff
@@ -0,0 +1,51 @@
+# VM details - boot order
+
+## Entering edit mode
+![VM details - Overview tab](img/B1-0-0.jpg)
+From the VM details page, Overview tab, the user clicks the pencil icon to edit the boot order.
+
+## Removing source items
+![Modal - default](img/B1-1-0.jpg)
+The modal pops up, displaying the currently listed bootable items by order of boot.
+This order was automatically set on the VM creation. This list displays bootable disks and NICS only.
+
+At this point, all the bootable sources are listed and there is no available resources to add to the list.
+An available sources counter at the bottom on the list highlights that to the user.
+
+![Removing item - actions menu](img/B1-1-1.jpg)
+From the action menu of each item, the user can remove an item from the list adding it to the available sources inventory.
+
+![Removing item](img/B1-1-2.jpg)
+An 'Add source' dropdown will appear at the bottom of the list, as the last item.
+The available sources counter at the bottom of the list updates to reflect that there are available bootable items. 
+
+## Adding source items
+![Add available source](img/B1-2-0.jpg)
+The user clicks the 'add source' dropdown at the bottom of the list.
```

Comment:
In other designs we us a (+) icon for that, https://github.com/openshift/openshift-origin-design/blob/master/web-console/4.0-designs/environment/environment.md, can we use an icon here, same as in other designs ?